Editor/Helper/AddressablesImporterOdinInspectorDrawer.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace UnityAddressableImporter.Editor.Helper
+{
+    using System;
+    using System.Diagnostics;
+    using Object = UnityEngine.Object;
+
+    public class AddressablesImporterOdinInspectorDrawer : IDisposable
```

Comment:
This class could be renamed to `AddressableImporterXXX` (singular) to keep consistent with other filenames, though using singular is a mistake in the first place.

Reply:
renamed to AddressablesImportersOdinHandler

Reply:
+